aspnet/identity/overview/features-api/customize_identity_model.md

```diff
@@ -16,19 +22,25 @@ ASP.NET Core Identity provides a framework for managing and storing user account
+
+1. Define or update a [data model in code](/ef/core/modeling/).
+1. Add a migration to translate this model into changes that can be applied to the database.
+1. Check that the migration correctly represents your intentions.
+1. Apply the migration to update the database to be in sync with the model.
+1. Repeat steps 1-4 to further refine the model and keep the database in sync.
 
-1. Define or update a [data model in code](https://docs.microsoft.com/en-us/ef/core/modeling/)
-2. Add a migration to translate this model into changes that can be applied to the database
-3. Check that the migration correctly represents your intentions
-4. Apply the migration to update the database to be in sync with the model
-5. Repeat steps 1-4 to further refine the model and keep the database in sync
+Migrations are added and applied using:
 
-Migrations should added and applied using the [Package Manager Console](https://docs.microsoft.com/en-us/ef/core/miscellaneous/cli/powershell) if you are using Visual Studio, or using the [dotnet commands](https://docs.microsoft.com/en-us/ef/core/miscellaneous/cli/dotnet) on the command line. ASP.NET Core also has a development-time error page handler that can be used to apply migrations when the application is run. For production applications it is often more appropriate to generate SQL scripts from the migrations and deploy these to the database as part of a controlled application and database deployment.
+* The [Package Manager Console](/ef/core/miscellaneous/cli/powershell) (PMC) if you are using Visual Studio.
+* The [dotnet commands](/ef/core/miscellaneous/cli/dotnet) on the command line.
+* Selecting the migrations link on the error page when the application is run.
 
-When a new application using Identity is created, steps 1 and 2 above have already been completed. That is, the initial data model already exists, and the initial migration has been added to the project. The initial migration still needs to be applied to the database, which can be done by running the `Update-Database` (PMC) or `dotnet ef database update` (.NET Core CLI) command, or by using the error page when the application is run.
+ASP.NET Core has a development-time error page handler that can be used to apply migrations when the application is run. For production applications, it is often more appropriate to generate SQL scripts from the migrations and deploy these to the database as part of a controlled application and database deployment.
 
-The series of steps defined above will need to be repeated as additional changes are made to the model as described in this article.
+When a new application using Identity is created, steps 1 and 2 above have already been completed. That is, the initial data model already exists, and the initial migration has been added to the project. The initial migration still needs to be applied to the database.
```

Comment:
Please add back in the commands to actually do this, otherwise people will struggle to find them.

Reply:
@ajcvickers  do you want me to add back 
which can be done by running the Update-Database (PMC) or dotnet ef database update (.NET Core CLI) command, or by using the error page when the application is run.
?

Reply:
Yes, please.